frontend/UserList.vue

```diff
@@ -39,6 +41,23 @@
               <p><label for="admin">Admin: </label><input class="form-control" type="checkbox" v-model="currentUser.admin" id="admin" /></p>
               <p><label for="disabled">Disabled: </label><input class="form-control" type="checkbox" v-model="currentUser.disabled" id="disabled" /></p>
             </form>
+
+            <p style="margin-top: 20px;"><h3 class="text-center">Roles</h3></p>
+            <form action="" id="editUserRolesForm" v-if="currentUser.id">
+              
+              <p>
+                <label for="admin_cb">Admin</label>
+                <input class="form-control" type="checkbox" id="admin_cb" value="admin" @click="updateUserRoleModal('admin')" v-model="currentUser.roles">
+              </p>
+              <p>
+                <label for="org_cb">Organizer</label>
+                <input class="form-control" type="checkbox" id="org_cb" value="organizer" @click="updateUserRoleModal('organizer')" v-model="currentUser.roles">
```

Comment:
Seems weird to use v-model to associate three different elements with the same data value. How does that work?

Reply:
Nevermind, just read https://vuejs.org/v2/guide/forms.html#Checkbox

Reply:
@mdempsky I was actually very impressed by Vue's feature/behavior for checkboxes using `v-model`.